Remotion/Core/UnitTests/Utilities/TypeUtilityTests.cs

```diff
@@ -121,21 +121,21 @@ public void TestAbbreviatedClosedGenericTypeWithPartiallyAbbreviatedTypeArgument
     [Test]
     public void TestGetType ()
     {
-      Type t = TypeUtility.GetType ("Remotion.UnitTests::Utilities.TypeUtilityTests", true);
+      Type t = TypeUtility.GetType ("Remotion.UnitTests::Utilities.TypeUtilityTests", true)!;

```

Comment:
should not be needed after Core.Core annotations are done. here and below.

Reply:
Unfortunately those are still needed, as there is currently no way to link a `bool` parameter with the nullability of the return type.